indico/web/client/styles/partials/_main.scss

```diff
@@ -11,26 +11,63 @@
 
 /* Styles for the main menu */
 
-.toolbar.global-menu {
-  @include font-family-title-light();
```

Comment:
Any reason for removing this (which changes the font? If it needs to be larger fine, but new font looks strange there.

I tried adding it back locally, and IMHO it looks better (master vs your PR + this line added back):

![image](https://github.com/indico/indico/assets/179599/f365affd-d788-4d0b-9f8b-0d05ed4f0ae4)
![image](https://github.com/indico/indico/assets/179599/7282ba95-e2ef-417a-a612-6bb68694964f)

Reply:
So the idea is to first bring all styling under the DS, and then improve the design afterwards.

> but new font looks strange there.

It's not the new font. It's just inheriting it from one of the ancestors. I want to flush out SUI first before I address font-related stuff.

Reply:
Fair enough. I should note that Roboto Light is most certainly not the appropriate font for this use case. In fact, any "Light" font is not a good use case for normal size text (and below) in general. You may use it on larger headings, but that's a different use case. Not all font rendering engines are made the same, and light fonts generally tend to render poorly on anything that isn't a Mac.

All in all we'll need to address the font choices anyway at some point. My long-term plan after the migration to DS is complete is to pick a good font set that works well across the board both in terms of readability and also from the aesthetic perspective.

Reply:
Sure, but IMHO those "more invasive" changes are better to keep for later since we can do all the nice improvements at first without exposing users to big "oh wth why does this look completely different now?!?!" surprises :)

Reply:
> "oh wth why does this look completely different now?!?!"

This is called a "canoe" problem. It's a temporary distraction that does not have no serious impact on usability. We generally shouldn't upgrade canoe problems into serious failures. ;)